docs-ref-services/latest/servicebus.md

```diff
@@ -15,402 +15,37 @@ ms.service: service-bus
-```
+### Version 7.x.x
 
-The event content is the event message or JSON-encoded string that contains multiple messages.
+To send and receive messages from an Azure Service Bus queue, topic or subscription, you would use the latest version of the `azure-servicebus`. This also allows to manage your Azure Service Bus resources like queues, topics, subscriptions and rules, but not the namespace itself.
 
-### Advanced features
+| Library | Reference | Samples | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    [azure-servicebus](https://pypi.org/project/azure-servicebus/)    |    [Reference](https://docs.microsoft.com/python/api/overview/azure/servicebus-readme?view=azure-python)    |    [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples)   |    [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus)    |
 
-#### Broker properties and user properties
+### Version 0.50.x
 
-This section describes how to use Broker and User properties defined [here](https://docs.microsoft.com/rest/api/servicebus/message-headers-and-properties):
+The older verson allows you to send and receive messages from an Azure Service Bus queue, topic or subscription, but it lacks a lot of the new features and performance improvements available in the latest version of the same package.
 
-```python
-sent_msg = Message(b'This is the third message',
-                   broker_properties={'Label': 'M3'},
-                   custom_properties={'Priority': 'Medium',
-                                      'Customer': 'ABC'}
-            )
-```
+The source code for the Azure Servicebus client libraries for Python is available on [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/servicebus_v0.50.3/sdk/servicebus/azure-servicebus/).
 
-You can use datetime, int, float or boolean
+| Library | Reference | Samples | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    [azure-servicebus](https://pypi.org/project/azure-servicebus/0.50.3/)   |    [Reference](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/0.50.3/index.html)    |    [Samples](https://github.com/Azure/azure-sdk-for-python/tree/servicebus_v0.50.3/sdk/servicebus/azure-servicebus/samples)    |    [GitHub](https://github.com/Azure/azure-sdk-for-python/tree/servicebus_v0.50.3/sdk/servicebus/azure-servicebus/)    |
```

Comment:
Consider adding "v0.50", otherwise the link texts for this and the previous table would look exactly the same since it is the same package.
Also, consider having more thank single words in link texts as it helps with accessibility. Screenreader users use this feature of looping through all links on the page and it is the link text that gets read out. Having more context in link text helps to differentiate between links

```suggestion
|    [azure-servicebus v0.50](https://pypi.org/project/azure-servicebus/0.50.3/)   |    [API Reference for azure-servicebus 0.50](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-servicebus/0.50.3/index.html)    |    [Samples for azure-servicebus 0.50](https://github.com/Azure/azure-sdk-for-python/tree/servicebus_v0.50.3/sdk/servicebus/azure-servicebus/samples)    |    [Source code for azure-servicebus 0.50](https://github.com/Azure/azure-sdk-for-python/tree/servicebus_v0.50.3/sdk/servicebus/azure-servicebus/)    |
```